Practice2/src/styles/components/common/items.css

```diff
@@ -19,21 +21,69 @@
 }
 
 .item-image-primary {
-  float: left;
   width: 40px;
   height: 40px;
 }
 
-.item-text-primary {
+.item-text {
   font-size: var(--fs-default);
   line-height: var(--lh-medium);
   color: var(--c-dark);
+  padding-left: var(--p-success);
+}
+
+.item-text-primary {
   text-transform: uppercase;
 }
 
 .item-text-secondary {
+  margin-bottom: var(--m-base);
+}
+
+.item-text-success {
   line-height: var(--lh-minimum);
   font-size: var(--fs-default);
   color: var(--c-dark);
   text-align: left;
+  margin-bottom: var(--m-default);
+  padding: 0 var(--p-base) 0 var(--p-success);
+  width: 351px;
```

Comment:
I still see the width here. You should find a better way for replacing this value.